lib/browser/property-descriptor.ts

```diff
@@ -23,24 +230,32 @@ export function propertyDescriptorPatch(_global: any) {
   if (canPatchViaPropertyDescriptor()) {
     // for browsers that we can patch the descriptor:  Chrome & Firefox
     if (isBrowser) {
-      patchOnProperties(window, eventNames.concat(['resize']));
+      patchOnProperties(window, eventNames, Object.getPrototypeOf(window));
```

Comment:
@mhevery , the reason is in IE/Edge, the `onProp` don't exist in `window` but in `WindowPrototype`.

The code below describe the issue.

```javascript
const desc = Object.getOwnPropertyDescriptor(window, 'onload');
// in IE
console.log(desc); // will be undefined.
// in chrome/firefox
console.log(desc); // will be {configurable:true,get/set....}
```

So in new `patchProperty` method, we only patch the prop which exist in the target object, so I pass the prototype of `window` for the existency check.